### ISSUE TITLE
Notification on OSX using wrong app icon

### ISSUE THREAD

**Issue by JohnMaguire** [sieren/QSyncthingTray#48]
Hi, and thank you again for this app. :) I got a couple friends using it now as well.

This isn't a high priority bug by any means, but I added QSyncthingTray to my "Login Items". After launching, it seemed to use the icon of Radiant Player for its notifications. I'm guessing it's probably dependent on what launched before it or something, but here's a screenshot from the notification panel.

<img width="336" alt="screen shot 2015-12-23 at 10 11 19 am" src="https://cloud.githubusercontent.com/assets/440033/11978998/c7bbdf7c-a95d-11e5-8678-0b9087a77913.png">

Thanks!

**Comment by sieren** [comment]
http://stackoverflow.com/questions/30881949/change-the-default-icon-for-qsystemtrayicon-notifications
Can you verify if it shows up correctly there?

**Comment by JohnMaguire** [comment]
Interestingly, I don't see QSyncthingTray in the Notifications preferences at all. I do, however, see Radiant Player listed twice.

**Comment by sieren** [comment]
Interesting indeed. I've seen this happen once or twice here, potentially related by Launch Items as well.
Out of curiousity: Did you build the app yourself at some point or have you been using the binary provided here?

https://discussions.apple.com/thread/5205059?start=0&tstart=0

**Comment by JohnMaguire** [comment]
I used the provided binary. :) In the Applications folder, the icon is correct.

**Comment by sieren** [comment]
On a short note, I've seen this too now recently on another app. QSyncThing hijacked the Icon of another Qt Built app... will have a look in their Bug database soon whether that one's on me or if its a known issue.

**Comment by ttimasdf** [comment]
+1, I've downloaded the release from the releases page, and extract them to two OSX desktops. Weirdly the notification icon appears good in one computer but show another app's icon constantly in the other. wondering what's happening..

**Comment by sieren** [comment]
Yeah, I'm not sure if this is a Qt bug or a QST bug. I will look into when I pick up work for the next version
